packages/ember-testing/lib/events.js

```diff
@@ -1,26 +1,28 @@
-import { jQuery } from 'ember-views';
 import { run } from 'ember-metal';
+import { assign } from 'ember-utils';
 
 const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
+const FORM_CONTROL_TAGS = ['INPUT', 'BUTTON', 'SELECT', 'TEXTAREA'];
 
 export function focus(el) {
   if (!el) { return; }
-  let $el = jQuery(el);
-  if ($el.is(':input, [contenteditable=true]')) {
-    let type = $el.prop('type');
+  if (el.isContentEditable || FORM_CONTROL_TAGS.indexOf(el.tagName) > -1) {
+    let type = el.getAttribute('type');
     if (type !== 'checkbox' && type !== 'radio' && type !== 'hidden') {
       run(null, function() {
-        // Firefox does not trigger the `focusin` event if the window
-        // does not have focus. If the document doesn't have focus just
-        // use trigger('focusin') instead.
+        el.focus();
 
-        if (!document.hasFocus || document.hasFocus()) {
-          el.focus();
-        } else {
-          $el.trigger('focusin');
-        }
+        let browserIsNotFocused = document.hasFocus && !document.hasFocus();
```

Comment:
Looks like there is a better, but similar sol'n [here](https://github.com/emberjs/ember.js/pull/16056/files#diff-f5610c59ca1ca80522629d553a3415c9R57)